packages/frontend/components/ImageBoard.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { useState, useRef, useEffect } from 'react'
+import { v4 as uuidv4 } from 'uuid'
+
+import ImageList from './ImageList'
+
+export default function ImageBoard() {
+  const [images, setImages] = useState([])
+  const imageCreatorRef = React.useRef<HTMLInputElement>(null)
+  const imageUrlRef = React.useRef<HTMLInputElement>(null)
+
+  function uploadImage(e) {
+    const imageCreator: string = imageCreatorRef.current.value
+    const imageUrl: string = imageUrlRef.current.value
+
+    if (imageCreator === '' || imageUrl === '') return
+    setImages((prevImages) => [
+      ...prevImages,
+      {
+        id: uuidv4(),
+        creator: imageCreator,
+        content: imageUrl,
+      },
+    ])
+    imageCreatorRef.current.value = null
+    imageUrlRef.current.value = null
+  }
```

Comment:
We really don't need all these upload forms, they will be replaced anyways. We only need the components right now, and this is just cluttering the codebase.

Reply:
removed file